html/changelogs/Intichaplain.yml

```diff
@@ -0,0 +1,4 @@
+author: Intigracy
+delete-after: True
+changes: 
+- rscdel: "Removes the message that everyone sees when the chaplain joins (which I added). Sounded good on paper, not so well in execution."
```

Comment:
literally what does this server war tribalism thing have to do with what he suggested